exercise1.py

```diff
@@ -0,0 +1,8 @@
+print("enter your age:", end=' ')
+age = int(input())
+if age == 18:
+    print('Congratulations')
```

Comment:
Please be consistent with your string quoting. Unless you have a good reason not to, stick to either `"` or `'`.
I recommend `"` only.